wpeverest/everest-forms
# Dev - Get gist integration

### All Submissions:


### Changes proposed in this Pull Request:

This PR introduces the new get gist integration.

### Dependency PR:
https://github.com/wpeverest/everest-forms-pro/pull/950
 
### How to test the changes in this Pull Request:

1. Generate the api key from the https://getgist.com/
2. Initiate the connection same as other integration from settings.
3. Go to the form setting add connection, map fields
4. After submission check the contact list.

### Types of changes:



### Other information:


### Changelog entry

> Dev  - Get gist integration.
